src/org/javarosa/core/util/PrefixTreeNode.java

```diff
@@ -73,19 +73,10 @@ public String toString () {
     }
 
     public String render() {
-        StringBuilder temp = new StringBuilder();
+        StringBuffer temp = new StringBuffer();
```

Comment:
This could stay, but either way is fine with me.

Reply:
Wouldn't that require adding the new `render` variant? Or do you mean using a `StringBuilder` in the zero-argument variant and never calling `render(StringBuffer buffer)`?

Reply:
Oh, you’re right of course. Disregard, please. I like your change as is.